src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+
+const useStyles = _theme => ({
+    root: {
+      maxWidth: 345,
+    },
+    media: {
+      height: 180,
+      width: 345
+    },
+    link: {
+        textDecoration: 'none'
+    }
+  });
+
+class CardVideo extends React.Component {
+    constructor(props){
+        super(props);
+        
+        this.classes =  props.classes;
+        this.videoInfo = props.videoInfo;
+        this.videoId = (typeof this.videoInfo.id === 'object') ? this.videoInfo.id.videoId : this.videoInfo.id;
```

Comment:
This comparison is not secure due to the variants of results of `typeof`.  Instead you could try:
```suggestion
        const  { id = {} } = props.videInfo;  // provide a default value - could be numeric too
        this.videoId = id.videoId || id; // if not an object with that videoId property, it takes the value provided for id or the default in your code
```